installer/win/BracketsConfigurator/BracketsConfigurator/BracketsConfigurator.cs

```diff
@@ -0,0 +1,67 @@
+using System;
```

Comment:
Copyright missing?

Reply:
@gavankar , please address this

Reply:
@nethip we have never had a copyright in the installer files so far(installer/win/Brackets.wxs), should we be adding one now?

Reply:
Yes please go ahead.

Reply:
@mbhavi, Please accept the following pull request which has this fix -
https://github.com/mbhavi/brackets-shell/pull/2

Reply:
@gavankar , I have merged your PR. @nethip , please find the review comments addressed here, as part of this merge.